<launch>

  <param name="Hand_Model" type="string" value="AG-2E"/>
  <param name="serial_port" type="string" value="/dev/ttyACM0"/>
  <param name="WaitDataTime" type="double" value="0.8"/>
  <node pkg="dh_hand_driver" type="hand_controller" name="hand_controller" output="screen">
  </node>

  <!-- 自己添加的， 修改了接口 -->
  <node pkg="dh_hand_driver" type="hand_controller_usr_client" name="dh_hand_driver_client"/>

</launch>
